rollup/overrides/sap/ui/fl/Change.js

```diff
@@ -0,0 +1,62 @@
+/*!
```

Comment:
In some file we have a copyright, in some files not - at the UI5 tooling we do not have one - but I assume it is the case because you copied the source for the overlay from the OpenUI5 codebase - so maybe it is better to keep the copyright for this one...